_posts/2017-08-31-Editathon.markdown

```diff
@@ -2,7 +2,7 @@
 layout: epic
 title: Art + Feminism and Artsy Wikipedia/Wikidata Editathon
 date: 2017-08-31
-categories: [Technology, emission, reaction, react-native, react, javascript]
+categories: [writing, culture, workshop]
```

Comment:
Wasn’t entirely sure how to categorise this one, but definitely not the previous ones :)

Reply:
Yeah, this is 👍